File-Formats/DocIO/FAQ.md

```diff
@@ -2044,330 +1133,179 @@ DocIO performs Mail merge by using the following methods:
 
 The following code example performs Mail merge by using the `Execute` method.
 
-{% tabs %}    
+{% tabs %}
 
 {% highlight c# tabtitle="C#" %}
-
-
 string dataBase = "Northwind.mdb";
-
 //Opens existing template.
-
 WordDocument doc = new WordDocument("Template.docx", FormatType.Docx);
-
 //Gets Data from the Database.
-
 OleDbConnection conn = new OleDbConnection("Provider=Microsoft.Jet.OLEDB.4.0;Data Source=" + dataBase);
-
 conn.Open();
-
 //Populates the data table.
-
 DataTable table = new DataTable();
-
 OleDbDataAdapter adapter = new OleDbDataAdapter("select * from employees", conn);
-
 adapter.Fill(table);
-
 adapter.Dispose();
-
 //Performs Mail Merge.
-
 doc.MailMerge.Execute(table);
-
 //Saves the document.
-
 doc.Save("Sample.docx", FormatType.Docx);
-
 //Closes the document.
-
 doc.Close();
-
-
-
 {% endhighlight %}
 
 {% highlight vb.net tabtitle="VB.NET" %}
-
 Dim dataBase As String = "Northwind.mdb" 
-
 ‘Opens the Word document.
-
 Dim doc As WordDocument = New WordDocument("Template.docx")
-
 ‘Creates database connection.
-
 Dim conn As OleDbConnection = New OleDbConnection("Provider=Microsoft.Jet.OLEDB.4.0;Data Source=" + dataBase)
-
 conn.Open()
-
 ‘Populates data table.
-
 Dim table As DataTable = New DataTable()
-
 Dim adapter As OleDbDataAdapter = New OleDbDataAdapter("select * from employees", conn)
-
 adapter.Fill(table)
-
 adapter.Dispose()
-
 ‘Performs Mail Merge.
-
 doc.MailMerge.Execute(table)
-
 ‘Saves the document.
-
 doc.Save("Sample.docx", FormatType.Docx)
-
 ‘Closes the document.
-
 doc.Close()
-
 {% endhighlight %}
 
-  {% endtabs %}
+{% endtabs %}
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Mail-Merge/Mail-merge-using-OleDbConnection).
 
-N> 
+N>
 For more information on Mail merge using DocIO, you can refer to online documentation link:
 [MailMerge](/File-Formats/DocIO/Working-with-MailMerge)
 
 ### Find and Replace
 
 This section illustrates how to perform a simple find and replace operation in a Word document by using Microsoft Office Automation and DocIO.
 
-Using Microsoft Office Automation
-
 The following code example illustrates how to search for a word in a Word document, replace it with another word and save the document under a new name.
```

Comment:
You have removed this heading line "Using Microsoft Office Automation"

No need to remove it.